src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -230,6 +231,16 @@ private void applyFilterExpression(FilterExpression filter) {
 
     protected abstract void stopLoadingAnimationIfApplicable(AppliedFilterEvent e);
 
+    private void showErrorStatus(FilterExceptionEvent e) {
+        if (e.panel != this) return;
```

Comment:
Actually, is the error-handling in applyStringFilter necessary? I think it wouldn't be used at all. (Not sure if Parser.parse will throw a FilterException.)